astropy/config/configuration.py

```diff
@@ -570,6 +575,64 @@ def get_config(packageormod=None, reload=False, rootname=None):
         return cobj
 
 
+def generate_astropy_config(filename=None):
+    """Generates a configuration file, from the list of ConfigItem
+    for each subpackage.
+
+    Parameters
+    ----------
+    filename : str or file object or None
+        If None, the default configuration path is taken from `get_config`.
+
+    """
+    import astropy
```

Comment:
I guess it wouldn't be a lot more work to generalize this part for any package (like `astroquery`) to use? If generalize, then I think it does make sense to live in `sphinx_astropy`. 🤔

Reply:
Yes, it should be pretty easy to generalize, I can have a look later.